dds/DCPS/Service_Participant.h

```diff
@@ -59,10 +59,10 @@ const char COMMON_DCPS_BIDIR_GIOP[] = "COMMON_DCPS_BIDIR_GIOP";
 const bool COMMON_DCPS_BIDIR_GIOP_default = true;
 
 const char COMMON_DCPS_BIT[] = "COMMON_DCPS_BIT";
-#ifdef DDS_HAS_MINIMUM_BIT
-const bool COMMON_DCPS_BIT_default = false;
-#else
+#if OPENDDS_CONFIG_BUILT_IN_TOPICS
 const bool COMMON_DCPS_BIT_default = true;
+#else
+const bool COMMON_DCPS_BIT_default = false;
```

Comment:
This could be a single line: `const bool COMMON_DCPS_BIT_default = OPENDDS_CONFIG_BUILT_IN_TOPICS;`
Which also means we can eliminate `COMMON_DCPS_BIT_default` and just use `OPENDDS_CONFIG_BUILT_IN_TOPICS` (unless it does something significant with type deduction so 0/1 would be different from false/true).